test/e2e/framework/configmanager/constants.go

```diff
@@ -39,16 +34,17 @@ const (
 	YKAdmCtrlName = "yunikorn-admission-controller-service" // YuniKorn Admission controller serivce name
 
 	// REST endpoints of YuniKorn
-	PartitionsPath   = "ws/v1/partitions"
-	QueuesPath       = "ws/v1/partition/%s/queues"
-	AppsPath         = "ws/v1/partition/%s/queue/%s/applications"
-	AppPath          = "ws/v1/partition/%s/queue/%s/application/%s"
-	ClustersPath     = "ws/v1/clusters"
-	NodesPath        = "ws/v1/partition/%s/nodes"
-	UserUsagePath    = "ws/v1/partition/%s/usage/user/%s"
-	GroupUsagePath   = "ws/v1/partition/%s/usage/group/%s"
-	HealthCheckPath  = "ws/v1/scheduler/healthcheck"
-	ValidateConfPath = "ws/v1/validate-conf"
+	PartitionsPath    = "ws/v1/partitions"
+	QueuesPath        = "ws/v1/partition/%s/queues"
+	AppsPath          = "ws/v1/partition/%s/queue/%s/applications"
+	AppPath           = "ws/v1/partition/%s/queue/%s/application/%s"
+	ClustersPath      = "ws/v1/clusters"
+	NodesPath         = "ws/v1/partition/%s/nodes"
+	UserUsagePath     = "ws/v1/partition/%s/usage/user/%s"
+	GroupUsagePath    = "ws/v1/partition/%s/usage/group/%s"
+	HealthCheckPath   = "ws/v1/scheduler/healthcheck"
+	ValidateConfPath  = "ws/v1/validate-conf"
+	FullStateDumpPath = "ws/v1/fullstatedump"
```

Comment:
The previous ykRest logs(queue/node/app) are replaced by fullstatedump and write to {specName}_ykFullStateDump.json".